build/update-attribution-files/make_attribution.sh

```diff
@@ -63,14 +63,16 @@ else
             # if the git_tags match across release branches, save the output state to avoid
             # rebuilding/regenerating
             if [[ $TARGET == *"checksums"* ]]; then
+                echo "Copying $LAST_RELEASE_BRANCH CHECKSUMS to $release"
                 mkdir -p $PROJECT_ROOT/_output/$release
-                cp -rf $PROJECT_ROOT/_output/$LAST_RELEASE_BRANCH/bin $PROJECT_ROOT/_output/$release
+                sed 's/$LAST_RELEASE_BRANCH/$release/' $PROJECT_ROOT/$LAST_RELEASE_BRANCH/CHECKSUMS > $PROJECT_ROOT/$release/CHECKSUMS
             fi
+
             if [[ $TARGET == *"attribution"* ]]; then
+                echo "Copying $LAST_RELEASE_BRANCH ATTRIBUTION to $release"
                 mkdir -p $PROJECT_ROOT/_output/$release
-                cp -rf $PROJECT_ROOT/_output/$LAST_RELEASE_BRANCH/{attribution,LICENSES} $PROJECT_ROOT/_output/$release
-            fi
-            build::attribution::generate $release
+                cp -rf $PROJECT_ROOT/$LAST_RELEASE_BRANCH/*TTRIBUTION.txt $PROJECT_ROOT/$release
```

Comment:
We pull this same *TTRIBUTION trick in the [makefile](https://github.com/aws/eks-distro/blob/main/Common.mk#L590). This is to cover cases when there are multiple attribution files generated, like for etcd.